data/tutorials/language/1ms_00_modules.md

```diff
@@ -24,52 +24,52 @@ In OCaml, every piece of code is wrapped into a module. Optionally, a module
 itself can be a submodule of another module, pretty much like directories in a
 file system.
 
-When you write a program, let's say using the two files `amodule.ml` and
-`bmodule.ml`, each automatically defines a module named
-`Amodule` and a module named `Bmodule`, which provides whatever you put into the
+When you write a program, let's say using the two files `athens.ml` and
+`berlin.ml`, each automatically defines a module named
```

Comment:
Do they define modules or create modules (or do those both mean the same thing)?

Reply:
Defines is probably better.